### PR TITLE
cleanup prefix 'z'

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut

### CODE REVIEW

**Comment by ngaut** [comment]
Please add prefix for table data and meta data.

**Comment by siddontang** [comment]
PTAL

**Comment by ngaut** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

File: kv/iter.go [hunk 1]
```diff
@@ -105,7 +93,7 @@ func (iter *decodeKeyIter) Value() []byte {
 }
 
 func (iter *decodeKeyIter) Key() string {
-	return string(DecodeKey([]byte(iter.iter.Key())))
+	return string([]byte(iter.iter.Key()))
```
Comment (disksing) [inline kv/iter.go]: `decodeKeyIter` can be removed now.

**Comment by siddontang** [comment]
PTAL @disksing

**Comment by disksing** [comment]
LGTM

**Pull request merged**